trials/userId.js

```diff
@@ -12,25 +12,23 @@ const { baseStimulus } = require("../lib/markup/stimuli");
  * @param {boolean} config.USE_ELECTRON - USE_ELECTRON flag
  * @param {boolean} config.USE_MTURK - USE_MTURK flag
  * @param {Object} options
- * @param {number} options.duration - The trial duration in milliseconds.
- * @param {string} options.stimulus - Onscreen stimulus in HTML to be shown in the trial, if not set default text is empty. If the stimulus is not provided, message should be provided as a string.
- * @param {string} options.setIdMessage - Onscreen text for setting user id or for the input box to enter patient id.
- * @param {boolean} options.responseEndsTrial - True if the trial ends on response,false if the trial waits for the duration, by default false value.
- * @param {boolean} options.defaultPatientId - The patient id to show when requesting a patient ID, if not set default is empty.
+ * @param {number} options.duration - trial duration in milliseconds. (default: 1000)
+ * @param {string} options.stimulus - Onscreen stimulus in HTML to be shown in the trial, if not set default text is empty. If the stimulus is not provided, message should be provided as a string. (default: "")
+ * @param {string} options.setIdMessage - Onscreen text for setting user id or for the input box to enter patient id. (default: "")
+ * @param {string} options.defaultPatientId - The patient id to show when requesting a patient ID. (default: "")
  */
 
 module.exports = function (jsPsych, config, options) {
   const defaults = {
-    setIdMessage: "",
+    duration: 1000,
```

Comment:
does userId need a duration? shouldn't it just ask for feedback until someone types something and hits okay? Additionally, shouldn't response ends trial be true?

Reply:
for MTURK the original code seems to need trial duration and response ends trial as false

Reply:
should that split on the `USE_MTURK` then?  the userID for non-MTURK is a text box that needs to be filled in and confirmed with a button I think? This function should handle both of those